# keep this - else the build fails on the package cluster
# see http://beefy5.nyi.freebsd.org/data/120i386-default/498628/logs/py27-pyside2-5.12.2.log
# I cannot reproduce the failure locally -- cmt
--- PySide2/QtWebEngineWidgets/CMakeLists.txt.orig	2019-03-21 14:37:54 UTC
+++ PySide2/QtWebEngineWidgets/CMakeLists.txt
@@ -25,6 +25,7 @@ set(QtWebEngineWidgets_include_dirs
                             ${Qt5Widgets_INCLUDE_DIRS}
                             ${Qt5Network_INCLUDE_DIRS}
                             ${Qt5WebChannel_INCLUDE_DIRS}
+                            ${Qt5WebEngineCore_INCLUDE_DIRS}
                             ${Qt5WebEngineWidgets_INCLUDE_DIRS}
                             ${libpyside_SOURCE_DIR}
                             ${QtCore_GEN_DIR}
@@ -45,7 +46,7 @@ set(QtWebEngineWidgets_libraries      pyside2
                             ${Qt5Core_LIBRARIES}
                             ${Qt5PrintSupport_LIBRARIES}
                             )
-set(QtWebEngineWidgets_deps QtGui QtWidgets QtNetwork QtWebChannel QtPrintSupport)
+set(QtWebEngineWidgets_deps QtGui QtWidgets QtNetwork QtWebChannel QtPrintSupport QtWebEngineCore)
 create_pyside_module(NAME QtWebEngineWidgets
                      INCLUDE_DIRS QtWebEngineWidgets_include_dirs
                      LIBRARIES QtWebEngineWidgets_libraries
